debian/changelog

```diff
@@ -1,3 +1,10 @@
+matrix-synapse-py3 (1.66.0~rc1+nmu1) UNRELEASED; urgency=medium
+
+  * Update debhelper to compatibility level 12.
+  * Drop the preinst script stopping synapse.
+
+ -- Jörg Behrmann <behrmann@physik.fu-berlin.de>  Thu, 25 Aug 2022 22:34:00 +0100
+
 matrix-synapse-py3 (1.66.0~rc1+nmu1) UNRELEASED; urgency=medium
 
```

Comment:
these two entries need to be combined. There should be at most one UNRELEASED version.

Reply:
```suggestion
```